ec.config.mjs

```diff
@@ -0,0 +1,8 @@
+import { defineEcConfig } from 'astro-expressive-code';
+
+export default defineEcConfig({
+  themes: ['material-theme-palenight'],
```

Comment:
<img width="1140" alt="image" src="https://github.com/user-attachments/assets/0ee0561c-a144-4a4a-873d-f047d8048475">

Reply:
[Dev site is deployed here](http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/)

Reply:
@tom-blake is there a way to set a max height? 
Some of the code blocks are really long - here is an example: http://centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/api/payment-requests/#list-payment-activities-for-a-merchant